src/project/qml/MuseScore/Project/internal/ScoresPage/ScoreItem.qml

```diff
@@ -384,4 +374,29 @@ FocusScope {
             }
         }
     }
+    
+    MouseArea {
```

Comment:
The MouseArea needs to be below (i.e. lower z-index) the CloudScoreIndicatorButton. Otherwise, the CloudScoreIndicatorButton is not hoverable anymore. What is the reason that you moved the MouseArea to the bottom of the file and thus increased its z-index?